microservice-transaction-sample/README.md

```diff
@@ -1,225 +1,183 @@
-# Microservice Transaction Sample
+# Create a Sample Application That Supports Microservice Transactions
 
-This tutorial describes how to create a sample application for Microservice Transaction that uses Two-phase Commit Transactions in ScalarDB.
-You can find more information about Two-phase Commit Transactions in ScalarDB [here](https://github.com/scalar-labs/scalardb/tree/master/docs/two-phase-commit-transactions.md).
+This tutorial describes how to create a sample application that supports microservice transactions in ScalarDB.
 
-## Prerequisites
-
-- Java (OpenJDK 8 or higher)
-- Gradle
-- Docker, Docker Compose
+## Overview
 
-## Sample application
+The sample e-commerce application shows how users can order and pay for items by using a line of credit. The use case described in this tutorial is the same as the [basic ScalarDB sample](../scalardb-sample) but takes advantage of [two-phase commit transactions](https://github.com/scalar-labs/scalardb/tree/master/docs/two-phase-commit-transactions.md) when using ScalarDB.
 
-### Overview
+The sample application has two microservices called *Customer Service* and *Order Service* based on the [database-per-service pattern](https://microservices.io/patterns/data/database-per-service.html).
```

Comment:
Ah, thanks for pointing that out. There's really no right or wrong in this case (some companies use "the" before proper nouns and some companies don't), but we should follow one or the other styles.

Since we made the distinction to use "*the* Customer Service" and "*the* Order Service" before in the Spring Boot sample doc (sounds more natural this way), I've updated this doc to reflect that style and made note of this for future reference.